src/models/service/balances.rs

```diff
@@ -0,0 +1,10 @@
+use crate::providers::info::TokenInfo;
+use serde::Serialize;
+
+#[derive(Serialize, Debug, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Balance {
+    pub token_info: TokenInfo,
+    pub amount: String,
```

Comment:
No strong preference, but would it make sense to use `balance` instead of `amount` to be in line with the transaction service? 

(same for `fiat_amount` vs `fiat_balance`)